doc/1.4/language.md

```diff
@@ -1505,6 +1516,24 @@ integer members (and arrays of similar) are translated
 to endian integers (or arrays of such) of similar size,
 with endianness matching the layout. Layout and endian integer
 members are accessed normally.
+
+The *discard identifer* "\_" may be used as the name of any number of members
```

Comment:
Inconsistent naming "\_" vs `_`, seems that you prefer the former in the context of declarations and the latter in the context of lvalues? why?

Reply:
i was inconsistent because i was inconsistent. and cause i wrote the docs for the discard identifier after i wrote them for the discard ref.

Really, using "`_`" should always be preferred, as depending on the setting a ticked underscore may not appear much different from surrounding text.

Reply:
I think plain `_` should be ok. If the user cares about seeing differences between identifiers and other words, then they should set things up accordingly.